src/components/Entities/EntityExecutionsBarChart.tsx

```diff
@@ -117,13 +117,16 @@ export const EntityExecutionsBarChart: React.FC<EntityExecutionsBarChartProps> =
         }
     );
 
-    const handleClickItem = React.useCallback(item => {
-        if (item.metadata) {
-            onToggle(item.metadata.name);
-            // const executionId = item.metadata as WorkflowExecutionIdentifier;
-            // history.push(Routes.ExecutionDetails.makeUrl(executionId));
-        }
-    }, []);
+    const handleClickItem = React.useCallback(
+        item => {
+            if (item.metadata) {
+                onToggle(item.metadata.name);
+                // const executionId = item.metadata as WorkflowExecutionIdentifier;
```

Comment:
Should we remove this?

Reply:
I haven't really changed this one :) So wasn't sure why we were keeping this comments. Will remove.

Reply:
done